app/Modules/Newsletter/Model/NewsletterService.php

```diff
@@ -66,7 +67,8 @@ class NewsletterService
 	public $linkGenerator;
 
 	/** Path to css file used for css inline of newsletter texts html */
-	const CSS_FILE_PATH = __DIR__ . DIRECTORY_SEPARATOR . DIRECTORY_SEPARATOR . 'templates' . DIRECTORY_SEPARATOR . 'Newsletter' . DIRECTORY_SEPARATOR . 'build.css';
+	const CSS_FILE_PATH = __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'Presenters' . DIRECTORY_SEPARATOR .
```

Comment:
Or just use method like `FileSystem::normalizePath()`. That's what I did in ApiGen for all paths possible.

Makes usage bulletproof for any OS programmer :)